src/builder/linker/linux/ld.cc

```
@@ -30,12 +31,25 @@ void Linker::constructLinkerArgs(std::string& input, std::string& output, std::v
   if (ctx->withStd) {
     // TODO: check if this works for all platforms
     linkerArgs.push_back("-dynamic-linker");
-    linkerArgs.push_back("/lib64/ld-linux-x86-64.so.2");
+
+    fs::path ld_linux_path;
+
+    void* handle = dlopen("ld-linux-x86-64.so.2", RTLD_LAZY);
+    if (!handle) { DEBUG_CODEGEN("Error getting library path: %s", dlerror()); }
```

Nice! This is really cool! Instead of DEBUG_CODEGEN we should raise an error instead, I can do that once I arrive at home since DEBUG_CODEGEN does not exist in release builds